recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "5.8" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha1: f0a5119bef4e1494e1af3871c288a1fe8200f304
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - snappy
+
+  run:
+    - snappy
```

Comment:
Please change `snappy` to [`snappy 1.1.6|1.1.7`]( https://github.com/conda-forge/conda-forge.github.io/blob/b44a39dbcd919222dee6cae3d7ee82bdf44986ec/scripts/pin_the_slow_way.py#L85 ) here as well.